src/kbmod/search/KBMOSearch.cpp

```diff
@@ -59,39 +58,40 @@ void KBMOSearch::setDebug(bool d) {
 }
 
 void KBMOSearch::enableCorr(std::vector<float> pyBaryCorrCoeff) {
```

Comment:
`pyBaryCorrCoeff`? (:D what does this name even mean why `py`, if it's a vector of floats, hungarian can really be unfriendly syntax in a strongly typed languages)

Reply:
I think I have fixed all of the py and made the casing consistent here.